moduleroot/Makefile.vars.mk.erb

```diff
@@ -10,6 +10,8 @@ COMPONENT_NAME ?= $(shell basename ${PWD} | sed s/component-//)
 compiled_path   ?= compiled/$(COMPONENT_NAME)/$(COMPONENT_NAME)
 root_volume     ?= -v "$${PWD}:/$(COMPONENT_NAME)"
 compiled_volume ?= -v "$${PWD}/$(compiled_path):/$(COMPONENT_NAME)"
+commodore_args  ?= --search-paths ./dependencies --search-paths .
+
```

Comment:
There's a whitespace change, one newline too much, between this and the `commodore` patch.